### PR TITLE
[GUI] Show the window when double clicking the taskbar icon

### PR DESCRIPTION
**Pull request by furszy**
Straightforward, ported #12838 solution for Windows users.

### CODE REVIEW

File: src/qt/pivx/pivxgui.cpp [hunk 1]
```diff
@@ -337,6 +337,9 @@ void PIVXGUI::changeEvent(QEvent* e)
             if (!(wsevt->oldState() & Qt::WindowMinimized) && isMinimized()) {
                 QTimer::singleShot(0, this, &PIVXGUI::hide);
                 e->ignore();
+            } else if((wsevt->oldState() & Qt::WindowMinimized) && !isMinimized()) {
```
Comment (Fuzzbawls) [inline src/qt/pivx/pivxgui.cpp:340]: nit: missing space after `if`

**Review by Fuzzbawls — APPROVED** [review]
utACK with a simple nit

File: src/qt/pivx/pivxgui.cpp (hunk 1 above)
Comment (furszy) [reply src/qt/pivx/pivxgui.cpp]: pushed

**Review by Fuzzbawls — APPROVED** [review]
re-utACK f8764b9e087849e04c62196ee00c849f4d121373

**Review by random-zebra — APPROVED** [review]
utACK f8764b9e087849e04c62196ee00c849f4d121373 and merging...

**Pull request merged**